CourseManager/Courses/Course 2 SwiftUICombineAndData/ContentView2.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ContentView.swift
+//  SwiftUICombineAndData
+//
+//  Created by Summer Crow on 2022-10-20.
+//
+
+import SwiftUI
+
+struct ContentView2: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
+    @State private var contentOffset = CGFloat(0)
+    
+    var body: some View {
+        NavigationView {
+            ZStack(alignment: .top) {
+                TrackableScrollView(offsetChanged: {
+                    offset in
+                    contentOffset = offset.y
+                }) {
+                    content
+                }
+                VisualEffectBlur(blurStyle: .systemMaterial)
+                    .opacity(contentOffset < -16 ? 1 : 0)
+                    .animation(.easeIn)
+                    .ignoresSafeArea()
+                    .frame(height: 0)
+            }
+            .frame(maxHeight: .infinity, alignment: .top)
+            .background(AccountBackground())
+            .navigationBarHidden(true)
+        }
+        .navigationBarHidden(true)
+        .navigationViewStyle(StackNavigationViewStyle())
+        .accentColor(colorScheme == .dark ? .white : Color(red: 0.12, green: 0.012, blue: 0.42))
+    }
+    
+    var content: some View {
```

Comment:
@sghourab do we need to keep nested views public?

Reply:
Need to make a habit of this

Reply:
updated.